pkg/genericapiserver/registry/generic/registry/store.go

```diff
@@ -67,6 +67,9 @@ type ObjectFunc func(obj runtime.Object) error
 //
 // TODO: make the default exposed methods exactly match a generic RESTStorage
 type Store struct {
+	// Copier is used to make some storage caching decorators work
```

Comment:
It's passed to decorators, you get to choose whether your decorator cares.  Since the type is controlled by the registry providing a copier from there seems reasonable.

Your decorator probably cares.